rpcs/blockService.go

```diff
@@ -240,6 +260,9 @@ func (bs *BlockService) ServeHTTP(response http.ResponseWriter, request *http.Re
 	if err != nil {
 		bs.log.Warn("http block write failed ", err)
 	}
+	bs.mu.Lock()
+	defer bs.mu.Unlock()
+	bs.memoryUsed = bs.memoryUsed - uint64(len(encodedBlockCert))
```

Comment:
Should we make these atomic instead of using locks?

Reply:
Oh I see that `rawBlockBytes()` already has a lock for it's duration and this is a short interval. Nevermind